config/mockconfig.go

```diff
@@ -212,13 +212,13 @@ func SetUpMockConfig(t *testing.T) error {
 
 	Data.DBConf = &DBConf{
 		Protocol:       "tcp",
-		Host:           "100.0.0.0",
-		Port:           "1234",
-		MaxIdleConns:   10,
-		MaxActiveConns: 4,
+		Host:           "ValidHost",
+		Port:           "ValidPort",
+		MinIdleConns:   2,
+		PoolSize:       4,
```

Comment:
why did we change the name from DBPoolSize

Reply:
Earlier we were using redigo package, which had MaxActiveConns. Now, as per your instructions, we are using go-redis, which has PoolSize in place of MaxActiveConns. So for making it similar to the package used I changed the config variable also.